sys/kern/tty.c

```diff
@@ -737,7 +738,12 @@ static void tty_drain_out(tty_t *tty) {
 }
 
 static int tty_vn_close(vnode_t *v, file_t *fp) {
-  /* TODO implement. */
+  tty_t *tty = v->v_data;
+  SCOPED_MTX_LOCK(&tty->t_lock);
+  assert(tty->t_opencount > 0);
+  tty->t_opencount--;
+  if (tty->t_opencount == 0 && tty->t_ops.t_notify_inactive)
```

Comment:
Please wrap calls to methods in following manner:

```c
static inline void tty_notify_inactive(tty_t *tty) {
  if (tty->t_ops.t_notify_inactive)
    tty->t_ops.t_notify_inactive(tty);
}
```

Reply:
Done